mmda/types/document.py

```diff
@@ -66,9 +66,6 @@ def annotate(
                 warnings.warn(f"The annotations is empty for the field {field_name}")
                 continue
 
-            # TODO: why is this good idea?
-            annotations = deepcopy(annotations)
```

Comment:
can u preserve a comment that there used to be this?  dont wanna just sweep it away wout doc